docs/reference/tags/form_tags/clear_cart/index.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: Clear Cart
+parent: Tags
+has_children: false
+---
+
+The Clear Cart Tag renders a button that clears the user's cart. 
+
+##### input
+{% raw %}
+```liquid
+{% form "clear_cart" %}
+	 <input type="submit" value="clear" {{id}}__clear" />
+ {% endform %}
+```
+{% endraw %}
+
+##### output
+{% raw %}
+```html
+<form action="/sites/cart" accept-charset="UTF-8" method="post">
+    <input type="hidden" name="_method" value="delete" autocomplete="off">
+    <input type="hidden" name="return_to" id="return_to" value="https://www.creator-website.com/book-tickets" autocomplete="off">
+    <input type="submit" value="Clear" class="button btn--outline cart-block__cart-clear">
+</form>
+```
+{% endraw %}
+
+##### redirect
+The Clear Cart Tag will redirect to the cart page by default.
+See the `return_to:` param to customise.
+
+##### cart ID
+The HTML input tag requires the id of the cart it is intended to clear.  
```

Comment:
Sorry I missed this in the first review, the cart ID isn't required

Reply:
Cool, thanks for confirming 🙌